Plugins/ExternalTexture/Source/ExternalTexture_Metal.mm

```diff
@@ -114,7 +114,9 @@
         { MTLPixelFormatDepth32Float,                   MTLPixelFormatInvalid                       }, // D32F
         { MTLPixelFormatStencil8,                       MTLPixelFormatInvalid                       }, // D0S8
     };
+  #ifndef TARGET_OS_OSX
```

Comment:
This fix doesn't seem right. If the array doesn't match, then there are going to be issues with the mapping. Do we need different tables depending on if it's OSX vs iOS?

Reply:
It seems to be, since that `BX_STATIC_ASSERT` throws error when it is not ignored by `#ifndef` or `//`.

Reply:
I know what is going on. This code actually doesn't compile in Xcode. We didn't catch it because our CI doesn't actually compile this plugin. I will fix this.